src/server/active_tran_server.hpp

```diff
@@ -78,7 +78,7 @@ class active_tran_server
     std::string m_ps_hostname;
     int m_ps_port = -1;
 
-    std::unique_ptr<page_server_conn_t> m_page_server_conn;
+    std::vector<std::unique_ptr<page_server_conn_t>> m_page_server_conn_vec;
```

Comment:
Even if not used anywhere, I would prefer a private typedef for `unique_ptr<page_server_conn_t>`. Makes code easier to read.
But I know @ribram does not.
Your choice. :)

Reply:
kept it the same way for now